core/modules/wmgr/python/dbMgr.py

```diff
@@ -359,19 +351,51 @@ def createTable(dbName):
             raise ExceptionResponse(500, "CSSError", "Failed to retrieve table schema from CSS", str(exc))
 
         # schema in CSS is stored without CREATE TABLE, so we are already OK
-
-    # create table
-    dbConn = Config.instance().dbConn()
-    try:
-        dbConn.createTable(tblName, schema, dbName)
-    except db.DbException as exc:
-        _log.error('Exception when creating table: %s', exc)
-        if exc.errCode() == db.DbException.TB_EXISTS:
-            raise ExceptionResponse(409, "TableExists", "Table %s.%s already exists" % (dbName, tblName))
-        raise
+        try:
+            dbConn.createTable(tblName, schema, dbName)
+        except db.DbException as exc:
+            _log.error('Exception when creating table: %s', exc)
+            if exc.errCode() == db.DbException.TB_EXISTS:
+                raise ExceptionResponse(409, "TableExists", "Table %s.%s already exists" % (dbName, tblName))
+            raise
 
     _log.debug('table %s.%s created succesfully', dbName, tblName)
 
+    if chunkColumns:
+        # Change table schema, drop _chunkId, _subChunkId, add chunkId, subChunkId
+
+        cursor = dbConn._conn.cursor()
+
+        try:
+            table = '`{0}`.`{1}`'.format(dbName, tblName)
+
+            # get current column set
+            q = "SHOW COLUMNS FROM %s" % table
+            cursor.execute(q)
```

Comment:
No, `execCommand0()` does not return anything :) But we could use `execCommandN()` here indeed. It's mostly historical, copied from the code which used cursor exclusively. This piece of code will have to be changed anyway when we switch to new `db` implementation based on sqlalchemy.